src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -47,21 +48,24 @@ static EnumInfo of(EnumMetaData enumMetaData) {
             }
         }
 
-        return new EnumInfo(enumClass.getName(), constants);
+        final String name = enumClass.getName();
+        return new EnumInfo(name, constants, docStrings.get(name));
     }
 
-    static EnumInfo of(String name, List<Object> constants) {
-        return new EnumInfo(name, constants);
+    static EnumInfo of(String name, List<Object> constants, Map<String, String> docStrings) {
+        return new EnumInfo(name, constants, docStrings.get(name));
```

Comment:
You use the dockey to fetch the docstring for FieldInfo only, won't these other places have collisions with same names in different packages? Or is this name fully qualified already?